src/components/Dropzone.svelte

```diff
@@ -211,6 +212,11 @@
           acceptedFiles.splice(0);
         }
 
+        // Files dropped keep input in sync
+        if (event.dataTransfer) {
+          inputRef.files = event.dataTransfer?.files;

```

Comment:
No need to have "?" at `event.dataTransfer?`.

Reply:
its already inside ```if (isEvtWithFiles(event))```

Reply:
its already inside if (isEvtWithFiles(event))

Reply:
Not exactly, it checks in `isEvtWithFiles` if files where received,

What I mean is
```suggestion
          inputRef.files = event.dataTransfer.files;

```
since the `if (event.dataTransfer) {` already promises that `event.dataTransfer` isn't `null` or `undefined`.

Reply:
ah right, the other way around - pretty sure I added this for some reason...

Reply:
removed 👍